spec/models/comment_spec.rb

```diff
@@ -500,9 +500,28 @@
       end
 
       context "direct parent is a tag" do
-        before { create(:comment, :on_tag, pseud: tag_wrangler.default_pseud) }
+        let!(:comment) { create(:comment, :on_tag, pseud: tag_wrangler.default_pseud) }
 
         include_examples "updates last wrangling activity"
+
+        it "notifies the tag wrangler when their comment on a tag is replied to" do
```

Comment:
I don't think these tests are in the right place. We're testing replies, so the direct parent is the comment we're replying to, not the tag. The tag is the ultimate parent.

I also think we can chain our matchers and avoid nesting an `expect` inside an `expect` block and checking `ActionMailer::Base` directly to make things a little tidier. Here's what I came up with: https://github.com/otwcode/otwarchive/commit/8deb1b19845c44585e8c1cb6d36895d82edd1e38